apps/api/jwt_auth/views.py

```diff
@@ -0,0 +1,36 @@
+from rest_framework import exceptions
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.views import APIView
+from rest_framework_jwt.authentication import JSONWebTokenAuthentication
+from rest_framework_jwt.compat import get_user_model
+
+from apps.api.models import APIKey
+
+
+class JWTKeyAuthentication(JSONWebTokenAuthentication):
+
+    def authenticate_credentials(self, payload):
+        """
+        Returns an active user that matches the payload's user id and email.
+        """
+        User = get_user_model()
+        try:
+            api_key = APIKey.objects.get(key=payload['iss'])
```

Comment:
Ah, yes. I think a manager function would be really helpful here. The `type=` filter was my attempt at making `APIKey` re-usable for other (future) types of keys so I should definitely be doing `APIKey.get_jwt_key()` or something.